### ISSUE TITLE
CC runtime install is not happening with latest operator code 

### ISSUE THREAD

**Issue by bpradipt** [confidential-containers/operator#117]
**Describe the bug**
CC runtime install is not happening with latest operator code 

**To Reproduce**
Just following the instructions in the README will hit the issue

**Describe the results you expected**
Successful install of the CC runtime on the K8s worker node

**Describe the results you received:**
No CC runtime install.

**Comment by bpradipt** [comment]
With recent changes, the daemonset gets created in `kube-system` namespace and fails due to missing service account
```
Events:
  Type     Reason        Age                  From                  Message
  ----     ------        ----                 ----                  -------
  Warning  FailedCreate  3s (x16 over 2m47s)  daemonset-controller  Error creating: pods "cc-operator-pre-install-daemon-" is forbidden: error looking up service account kube-system/cc-operator-controller-manager: serviceaccount "cc-operator-controller-manager" not found
```

The deployment manifest needs to be updated to align with the latest changes

**Comment by mythi** [comment]
> The deployment manifest needs to be updated to align with the latest changes

should we drop `deploy.yaml` and change the instructions to point to `config/default`?

**Comment by bpradipt** [comment]
@mythi that would make sense and avoid maintaining two different manifests.

As for people hitting the current issue and looking for a quick workaround they can update the ccruntime.yaml with namespace and that should be it.

```
diff --git a/config/samples/ccruntime.yaml b/config/samples/ccruntime.yaml
index 47f9d6b..94fec88 100644
--- a/config/samples/ccruntime.yaml
+++ b/config/samples/ccruntime.yaml
@@ -2,6 +2,7 @@ apiVersion: confidentialcontainers.org/v1beta1
 kind: CcRuntime
 metadata:
   name: ccruntime-sample
+  namespace: confidential-containers-system
 spec:
   # Add fields here
   runtimeName: kata
```

**Comment by mythi** [comment]
> As for people hitting the current issue and looking for a quick workaround they can update the ccruntime.yaml with namespace and that should be it.

The right fix would be to use `0.1.0` image instead of `:latest`. `CcRuntime` is a cluster scoped resource so I'm not sure how adding namespace there would help.

**Comment by liudalibj** [comment]
> > As for people hitting the current issue and looking for a quick workaround they can update the ccruntime.yaml with namespace and that should be it.
> 
> The right fix would be to use `0.1.0` image instead of `:latest`. `CcRuntime` is a cluster scoped resource so I'm not sure how adding namespace there would help.

I meet the same issue here, when I follow [this guide](https://github.com/confidential-containers/documentation/blob/v0.1.0/quickstart.md), it failed at [run-local.sh script](https://github.com/confidential-containers/operator/blob/main/tests/e2e/run-local.sh).
>...
> ccruntime.confidentialcontainers.org/ccruntime-sample created
> ERROR: cc-operator-daemon-install pod is not running
> DEBUG: Pod
> + kubectl describe pods/ '-n confidential-containers-system'
> error: arguments in resource/name form must have a single resource and name
> + true
> + kubectl logs pods/ '-n confidential-containers-system'
> error: arguments in resource/name form must have a single resource and name
> + true
> + set +x

After I update the [operator image](quay.io/confidential-containers/operator) tag to v0.1.0 at https://github.com/confidential-containers/operator/blob/v0.1.0/deploy/deploy.yaml#L3692, the issue is fixed.
```
root@peer-pod1:~/git/operator/config/samples# kubectl get po -A
NAMESPACE                        NAME                                              READY   STATUS    RESTARTS   AGE
confidential-containers-system   cc-operator-controller-manager-56bcbd94fd-nq5zh   2/2     Running   0          81m
confidential-containers-system   cc-operator-daemon-install-nlstl                  1/1     Running   0          3m39s
confidential-containers-system   cc-operator-pre-install-daemon-9rhzr              1/1     Running   0          3m42s
kube-flannel                     kube-flannel-ds-qhk8f                             1/1     Running   0          3h55m
kube-system                      coredns-6d4b75cb6d-7lbxw                          1/1     Running   0          3h55m
kube-system                      coredns-6d4b75cb6d-xb86s                          1/1     Running   0          3h55m
kube-system                      etcd-peer-pod1.fyre.ibm.com                       1/1     Running   0          3h56m
kube-system                      kube-apiserver-peer-pod1.fyre.ibm.com             1/1     Running   0          3h56m
kube-system                      kube-controller-manager-peer-pod1.fyre.ibm.com    1/1     Running   0          3h55m
kube-system                      kube-proxy-f5ws2                                  1/1     Running   0          3h55m
kube-system                      kube-scheduler-peer-pod1.fyre.ibm.com             1/1     Running   0          3h56m
root@peer-pod1:~/git/operator/config/samples#
```

@bpradipt maybe we should fix the image tag issue in deploy.yaml and retag [v0.1.0 operator codes](https://github.com/confidential-containers/operator/blob/v0.1.0) with the fix.

**Comment by fidencio** [comment]
This has been fixed by https://github.com/confidential-containers/operator/pull/123
